src/wiseagents/transports/stomp.py

```diff
@@ -80,13 +81,17 @@ def __getstate__(self) -> object:
         del state['_response_receiver']
         del state['_event_receiver']
         del state['_error_receiver']
+        del state['conn']
```

Comment:
I know conn and conn2 weren't introduced in this PR but might be good to update these with names that indicate their purpose.